content/docs/sidebar.json

```diff
@@ -75,7 +75,11 @@
         "slug": "sharing-data-and-model-files"
       },
       "shared-development-server",
-      "data-registries"
+      "data-registries",
+      {
+        "label": "CI and ML",
+        "slug": "ci-and-ml"
```

Comment:
Is is "and" or "for"?

Reply:
I had "and" in mind in the sense of storytelling; the story of how CI and ML play nicely together. It's a matter of taste, and "and" can be easily used and be more down to earth.

Reply:
I mean the title in the actual doc uses "for". Up to you which one but let's just use the same in nav vs doc please.

Reply:
Sure. My bad...